### ISSUE TITLE
accessibility id not found for the html elements rendered via react-native-webview in Appium Inspector

### ISSUE THREAD

**Issue by jakhil232** [react-native-webview/react-native-webview#3547]
**Issue** - In my react native app, while trying to inspect html content rendered through react-native-webview, via Appium Inspector, I am unable to find accessibility id's for the content displayed. Though, the same html when rendered in iOS, the accessibility id's are present there. In android, I am able to get the xpath & other details, but accessibility id's are not found. Do we need to add separate props or any other setting that needs to be done in order to access the testId's?

I am using Webview component as follows - 

<WebView
        source={{
          html,
        }}
        style={[webViewMarginStyle(), style.webViewOpacity]}
        webviewDebuggingEnabled={true}
/>

Any extra prop required or any javascript code that needs to be injected? This is a blocker for QA team, as they are unable to proceed with automation due to absence of testIds.

**Comment by jakhil232** [comment]
Hey community, any info over this?

**Comment by jakhil232** [comment]
Anyone know fix for this issue?

**Comment by ReinoldoMethod** [comment]
Hello, did you get a fix?

**Comment by meatnordrink** [comment]
[This thread](https://github.com/react-native-webview/react-native-webview/issues/1863) addresses a similar issue, and has a little more activity - might or might not address this, but worth checking out. 

Incidentally, we've had an inverse issue - ONLY test id's are available on Android, while text selectors work on iOS.
